sys/parsoid.js

```diff
@@ -281,26 +287,6 @@ class ParsoidService {
     generateAndSave(hyper, req, format, currentContentRes) {
         // Try to generate HTML on the fly by calling Parsoid
         const rp = req.params;
-        // Helper for retrieving original content from storage & posting it to
```

Comment:
This was commented out during the last transition, but we forgot to start using it again. It helps Parsoid making a quicker conversion for the same revid, so we should probably keep this and activate it after the current transition.

Either way, this change does not belong in this PR

Reply:
We will not resurrect it ever afaik after talking to Subbu. I will remove it from this PR though, you're right.